src/victory-portal/render-in-portal.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Log } from "../victory-util/index";
+
+export default class RenderInPortal extends React.Component {
+  static propTypes = {
+    children: React.PropTypes.node
+  };
+
+  static contextTypes = {
+    portalUpdate: React.PropTypes.func,
+    portalRegister: React.PropTypes.func,
+    portalDeregister: React.PropTypes.func
+  }
+
+  checkForPortal() {
+    if (!this.checkedContext) {
+      if (typeof this.context.portalUpdate !== "function") {
+        const msg = "`renderInPortal` is not supported outside of `VictoryContainer`. " +
+          "Component will be rendered in place";
+        Log.warn(msg);
+        this.renderInPlace = true;
+      }
+      this.checkedContext = true;
+    }
+  }
+
+  componentDidUpdate() {
+    this.checkForPortal();
```

Comment:
sure, but why would we need it on componentDidUpdate at all if it's on render?

(but do see my other comment, unless I'm overlooking something I don't think this can ever change during the lifecycle of the component, due to the way I wrote `getChildContext` in `VictoryContainer`)